recipe/build.sh

```diff
@@ -19,5 +21,20 @@ export INSTALL_INCLUDE="${PREFIX}/include"
 export BLAS="-lopenblas"
 export LAPACK="-lopenblas"
 
+# (optional) write out various make variables for easier build debugging
+eval ${LIBRARY_SEARCH_VAR}="${PREFIX}/lib" make config 2>&1 | tee make_config.txt
+
 eval ${LIBRARY_SEARCH_VAR}="${PREFIX}/lib" make -j1
+
+# make install below fails to link libmetis unless it is copied to $PREFIX/lib
+cp $SRC_DIR/lib/libmetis$DYNAMIC_EXT $PREFIX/lib
```

Comment:
Here is the relevant METIS section of the SuiteSparse Makefile
```
# configure and compile METIS, placing the libmetis.* library in
# SuiteSparse/lib and the metis.h include file in SuiteSparse/include.
metis: include/metis.h

# Install the shared version of METIS in SuiteSparse/lib.
# The SO_INSTALL_NAME commmand is only needed on the Mac, so *.dylib is
# hardcoded below.
include/metis.h:
ifeq (,$(MY_METIS_LIB))
	- ( cd metis-5.1.0 && $(MAKE) config shared=1 prefix=$(SUITESPARSE) )
	- ( cd metis-5.1.0 && $(MAKE) )
	- ( cd metis-5.1.0 && $(MAKE) install )
	- $(SO_INSTALL_NAME) $(SUITESPARSE)/lib/libmetis.dylib \
                             $(SUITESPARSE)/lib/libmetis.dylib
else
	@echo 'Using pre-installed METIS 5.1.0 library at ' '[$(MY_METIS_LIB)]'
endif
```
I think what we would want is prefix=$PREFIX so that it ends up installed where we would like as opposed to $SUITESPARSE ($SUITESPARSE is the same as $SRC_DIR)

Also, it seems like a bug in the Makefile that .dylib is listed there when the extension would be .so on linux?